### PR TITLE
Coaching companies Blog Post Part 2

### PR DESCRIPTION
**Pull request by anasofiapinho**
Coaching companies blog post without the pictures. Updated minor errors
on the previous blog post.

### CODE REVIEW

**Comment by carpodaster** [comment]
Hey @anasofiapinho, I think the images are those of the 1st CC blog post still :)

**Comment by carpodaster** [comment]
> Coaching companies blog post without the pictures

Ah, excuse me. I should have read the body of the PR… _wanders off to grab more coffee_

File: blog/_posts/2016-08-25-coaching-companies-part-2.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+(Atlanta, USA)
+
+![Mitpal at Elevator Factory](/img/blog/2016/rgsoc-2016-coaching-companies_rubycats-codequest.png)<font color="grey"><small><i>Sherri and Anitha at Elevator Factory! (Image: Sherri and Anitha)</i></small></font>
+
+### Sherri and Anitha
+
+*"The support that we have received in RGSoC and the Coaching Company is invaluable. We have an amazing team that has donated time and resources to help us be successful during RGSOC. Our co-working space was donated by the Elevator Factory, which has allowed us to come in and work together every day. Our coaches, Pamela Vickers and Pete Holiday, have given countless hours of their own time to help us work through any issues that we may be having. Without this support system in place, learning and accomplishing all that we have would be difficult at best. Our skill set along with our confidence continues to grow, and we know that our career goals are definitely within our reach."*
+
+
+## kind3d and [Namics](http://namics.com)
```
Comment (alicetragedy) [inline blog/_posts/2016-08-25-coaching-companies-part-2.md]: small typo -> kindr3d

**Comment by alicetragedy** [comment]
It looks good, @anasofiapinho — I'm looking forward to seeing the pictures 😍 
The text all sounds great, too, so I think it would be ready to go once you have all the pictures.
Quick note: it's usually good practice to encapsulate everything that fits together into a PR — in theory you could have made (and pushed) the changes to the old post separately, as they're not really required for the new blog post. Anyway, that's not a huge deal, let's keep it like this and merge all the changes together once the pictures are in! 👍

**Comment by anasofiapinho** [comment]
@carpodaster ahahah, no worries! I was going to push everything but Illustrator crashed and I lost everything that I had done. :/

**Comment by anasofiapinho** [comment]
@alicetragedy I am sorry, I had no idea! I just added those changes because when I was editing the new post I realised I had some parenthesis I needed to delete. Thanks for letting me know. I will try not to make the same mistake again. I have so much to learn about how to use GitHub!!!

**Pull request merged**